library/utils.py

```diff
@@ -0,0 +1,107 @@
+import csv
+import json
+import os
+import urllib2
+
+from django.core.files.base import ContentFile
+from django.utils.translation import ugettext as _
+
+OPENLIBRARY_API_URL = 'https://openlibrary.org/api/books?'
+
+
+def to_unicode(text):
+    """Do its best to return an unicode string."""
+    if isinstance(text, unicode):
+        return text
+    else:
+        try:
+            text = text.decode('utf-8')
+        except UnicodeDecodeError:
+            text = text.decode('latin-1')
```

Comment:
I don't understand: if it can't decode from `utf-8` it sure won't be able to decode from `latin-1` (which is a subset of `utf-8`).